lib/MetaCPAN/Web/Controller/Author.pm

```diff
@@ -50,15 +51,15 @@ sub index : Chained('root') PathPart('') Args(0) {
     my ( $author, $data ) = ( $author_cv->recv, $releases_cv->recv );
     $c->detach('/not_found') unless ( $author->{pauseid} );
 
-    my $faves_cv = $c->model('API::Favorite')->by_user( $author->{user} );
+    my $faves_cv = $c->model('API::Favorite')->by_user( $author->{pauseid} );
```

Comment:
Wondering about this change (and the other similar one).  Non-authors can also use the `++` functionality, so we need to make sure we're not breaking anything for them.